package.json

```diff
@@ -36,7 +36,6 @@
     "test": "ava test/*",
     "lint": "eslint --cache .",
     "checks": "npm test && npm run lint",
-    "refresh": "npm run clean && npm ci",
```

Comment:
Rationale?

Reply:
`npm ci` from within script fails on my machine, not willing to troubleshoot.

Reply:
And `npm i` does not?

Reply:
It works both on my machine and in CI. We can just as well leave it in. You don't need to use it.

Reply:
I mean, `npm ci` works, but `npm ci` invoked by `npm run refresh` does not. Does that work for you?

Reply:
Sure, just tried it. I also don't know why there would be a difference. What Linux version does your installation of WSL rely on?

Reply:
I'm on Windows Windows, no WSL. And Node 14.0.0, I should probably upgrade. Re-adding then.